modules/swagger-codegen/src/main/java/io/swagger/codegen/utils/ModelUtils.java

```diff
@@ -0,0 +1,29 @@
+package io.swagger.codegen.utils;
+
+import io.swagger.codegen.CodegenModel;
+
+import java.util.List;
+import java.util.Map;
+
+public class ModelUtils {
+    public static CodegenModel getModelByName(final String name, final Map<String, Object> models) {
```

Comment:
I could just add something like this but I thought it's pretty self explanatory  looking at  method signature.
   /**
     *
     * @param name  Name of the model
     * @param models Map of models
     * @return  Searches for the model by name in the map of models and returns it
     */